ci/nightly/pipeline.yml

```diff
@@ -44,7 +44,7 @@ steps:
     command: bin/ci-builder run stable bin/pyactivate --dev -m ci.test.build x86_64
     timeout_in_minutes: 60
     agents:
-      queue: builder
+      queue: builder-linux-x86_64
 
   - command: bin/ci-builder run stable bin/pyactivate --dev -m ci.nightly.trim_pipeline
     if: build.source == "ui"
```

Comment:
Should `feature-benchmark` below get an explicit queue? Not sure if that was intentional since it's such a long test? / I don't exactly understand how that test works 😅

Reply:
Oh it totally should! I just missed it. Thanks.